### ISSUE TITLE
Pass state fails validation when InputPath is used to access the Context Object

### ISSUE THREAD

**Issue by dalecampbell** [serverless-operations/serverless-step-functions#402]
# This is a Bug Report

## Description
In August 2020, AWS [released](https://aws.amazon.com/blogs/aws/aws-step-functions-adds-updates-to-choice-state-global-access-to-context-object-dynamic-timeouts-result-selection-and-intrinsic-functions-to-amazon-states-languages/) numerous updates for Step Functions, most of which you support. They added support for accessing the Context Object outside of just Parameters. See https://docs.aws.amazon.com/step-functions/latest/dg/input-output-contextobject.html

When using a Pass State with an InputPath to referencing the $$.Execution.Id the validator returns an error even though this properly deployed and works without issue. 

For bug reports:
* What went wrong? 
  * Pass state fails validation when InputPath is used to access the Context Object
* What did you expect should have happened? 
  * Validation should pass. When validation is turned off the SFN deploys without issue.
* What was the config you used?

name: test-pass-inputpath
```
definition:
  Comment: "Testing getting Context Object from InputPath in Pass State"
  StartAt: pass-execution-details
  States:
    pass-execution-details:
      Type: Pass
      ResultPath: $.AWS_STEP_FUNCTIONS_STARTED_BY_EXECUTION_ID
      InputPath: $$.Execution.Id
      End: True
```

* What stacktrace or error message from your provider did you see?

```
✕ State machine "test-pass-inputpath" definition is invalid:
  undefined: undefined
  No errors
```

## Additional Data
```
Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              13.12.0
     Framework Version:         2.4.0 (local)
     Plugin Version:            4.0.4
     SDK Version:               2.3.2
     Components Version:        3.2.1
```
```
serverless-step-functions@2.29.0
  └── asl-validator@1.9.3 
```

**Comment by dalecampbell** [comment]
Also opened an issue at asl-validator, i believe this is an issue with the validation plugin. https://github.com/ChristopheBougere/asl-validator/issues/65

**Comment by dalecampbell** [comment]
This issue was fixed with version 1.9.4 of the asl-validator.
